xml/osc_building.xml

```diff
@@ -20,8 +20,120 @@
  
  <sect1 xml:id="sec.obs.building.spec2rpm">
   <title>From Spec Files (RPM)</title>
-  <para/>
-  <remark>TBD</remark>
+  <para>
+   To create an RPM package you need:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     the spec file
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     a file with the extension <filename class="extension">.changes</filename>
```

Comment:
maybe explain the purpose of these files a bit? e.g. something like:
"a file named PACKAGE_NAME.changes that can be used to document the package history."
(this also goes for the other items)

Reply:
Good point! 👍 Will do so.